bigchaindb/backend/localmongodb/query.py

```diff
@@ -301,3 +303,15 @@ def get_validator_set(conn, height=None):
     )
 
     return list(cursor)[0]
+
+
+@register_query(LocalMongoDBConnection)
+def get_received_votes_for_election(conn, election_id, election_public_key):
```

Comment:
A more extensible approach might be to use `get_filtered_txids` and check for public keys outside. Otherwise, the database interface gets concerned with the election logic (in this case the logic of what is the election vote). What do you think?

Reply:
The only thing that makes this function a little specialized is including `operation: 'VALIDATIOR_ELECTION_VOTE'`. Do you want me to make that as a variable?

Reply:
`operation` is already a variable of `get_filtered_txids` btw. `get_received_votes_for_election` is "special" in a sense that different types of elections can have different ways of identifying their votes.